packages/react/src/themes/teams/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,9 @@
+export interface TreeTitleVariables {
+  defaultColor: string
```

Comment:
```suggestion
  color: string
```
https://github.com/stardust-ui/react/pull/1779/files#r319038491